docs/index.rst

```diff
@@ -38,8 +38,7 @@ NVIDIA DALI Documentation
 
    pipeline
    data_types
-   supported_ops
-   math
+   data_manip
```

Comment:
```suggestion
   data_manipulation
```
I don't think we need to shorten it here.

Reply:
How about something like:
```suggestion
.. toctree::
   :hidden:
   :caption: Data Manipulation
   graph_building # Put some links to Pipeline doc and Getting started here
   indexing
   math
   supported_ops
```

Reply:
To keep it top-level